### PR TITLE
get, getV, reorder for records --redux

### PR DESCRIPTION
**Pull request by eparejatobes**

### CODE REVIEW

**Comment by eparejatobes** [comment]
I vote forma making reorder ad-hoc for records (product types). I don't think there's an easy way of doing it, and I don't want to delay this further. Same for update, but maybe this one could work just using replace.

**Comment by eparejatobes** [comment]
I also added update based on replace from KLists. This should also close #74

**Comment by eparejatobes** [comment]
@laughedelic please review. I'm closing #74 in favor of this.

File: src/main/scala/cosas/records/syntax.scala [hunk 1]
```diff
@@ -4,37 +4,42 @@ import ohnosequences.cosas._, types._, fns._, klists._
...
+      new (RT := Vs)(repl(vs,s))
 
   }
 
   final case class RecordTypeSyntax[RT <: AnyRecordType](val rt: RT) extends AnyVal {
 
-    def apply[Vs <: AnyKList { type Bound = AnyDenotation }](values: Vs)(implicit
-      reorder: AnyApp1At[TakeFirst[RT#Raw], Vs]
-    ): RT := RT#Raw = rt := reorder(values)
+    def apply[Vs <: AnyKList.withBound[AnyDenotation], RTRaw <: RT#Raw](values: Vs)(implicit
```
Comment (laughedelic) [inline src/main/scala/cosas/records/syntax.scala]: If I understood https://github.com/ohnosequences/cosas/pull/76#issuecomment-154698086 right, you suggest to rename it to `reorder`. Fine for me.

**Comment by eparejatobes** [comment]
let's merge this!

**Comment by laughedelic** [comment]
ok :shipit:

**Pull request merged**